### ISSUE TITLE
qiime info --py-packages broken

### ISSUE THREAD

**Issue by thermokarst** [qiime2/q2cli#202]
**Bug Description**
The `info` subcommand accepts an optional `--py-packages` flag, which should print a list of installed packages. Supplying this flag raises an `AttributeError`:

```bash
Installed Python packages
Traceback (most recent call last):
  File "/Users/matthew/.conda/envs/qiime2-2018.11/bin/qiime", line 11, in <module>
    sys.exit(qiime())
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/q2cli/info.py", line 64, in info
    _echo_installed_packages()
  File "/Users/matthew/.conda/envs/qiime2-2018.11/lib/python3.5/site-packages/q2cli/info.py", line 45, in _echo_installed_packages
    for i in pip.get_installed_distributions()])
AttributeError: module 'pip' has no attribute 'get_installed_distributions'
```

**Steps to reproduce the behavior**
1. `qiime info --py-packages`

**Expected behavior**
The installed python packages should be displayed. Maybe this should just run `conda list`?

**Comment by ebolyen** [comment]
Yeah that was technically a private API in `pip`. I forgot about this, but we use a different mechanism in provenance for that reason:
https://github.com/qiime2/qiime2/blob/e938a76b615a13dbbd7ec9f32d9273885886013a/qiime2/core/archive/provenance.py#L204-L206

That being said, I like the idea of jumping straight to conda instead.

**Comment by ebolyen** [comment]
fixed in #212
